app/models/enrollment.rb

```diff
@@ -6,6 +6,10 @@ class Enrollment < ActiveRecord::Base
   has_one :certificate
 
   validates :honor_code, acceptance: true
+  validates :student_id, uniqueness: {
+                           scope: :course_id,
+                           message: 'You have already enrolled.'
+                         }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.